gremlin/templates/chao-service-account.yaml

```diff
@@ -14,7 +14,10 @@ rules:
     resources: ["replicasets", "deployments", "statefulsets", "daemonsets"]
     verbs: ["get", "watch", "list"]
   - apiGroups: [""]
-    resources: ["pods", "nodes", "services", "namespaces"]
+    resources: ["pods", "nodes", "services", "namespaces", "replicationcontrollers"]
+    verbs: ["get", "watch", "list"]
+  - apiGroups: ["apps.openshift.io"]
+    resources: ["deploymentconfigs"]
```

Comment:
References for `apiGroup` = `apps.openshift.io`...

https://bugzilla.redhat.com/show_bug.cgi?id=1886888
https://github.com/luckylittle/openshift-cheatsheet/blob/master/role-admin.yaml#L320-L325
https://forum.gitlab.com/t/openshift-clusterrole-for-gitlab-runner-sa/80252
https://www.ibm.com/docs/en/instana-observability/215?topic=requirements-installing-host-agent-openshift